src/geometry/shape_views/collider_view.rs

```diff
@@ -93,6 +94,7 @@ impl<'a> fmt::Debug for ColliderView<'a> {
             ColliderView::RoundConvexPolyhedron(view) => write!(f, "{:?}", view.raw),
             #[cfg(feature = "dim2")]
             ColliderView::RoundConvexPolygon(view) => write!(f, "{:?}", view.raw),
+            ColliderView::Custom(_) => write!(f, "Custom"),
```

Comment:
if I'm understanding correctly, a good information to add here would by the number identifying the shape ?

maybe `as_typed_shape()` would be interesting ?

```suggestion
            ColliderView::Custom(shape) => write!(f, "{:?}", shape.as_typed_shape()),
```

Reply:
This wouldn’t be very useful because as_typed_shape would just return TypedShape::Custom.

The thing is that we don’t know if the custom shape implements Debug.